packages/wonder-blocks-announcer/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@babel/runtime": "catalog:",
-    "@khanacademy/wonder-blocks-core": "^9.0.0"
+    "@khanacademy/wonder-blocks-core": "workspace:*"
```

Comment:
@marcysutton I found this while troubleshooting an unrelated changesets issue. Hope it is fine to switch to the latest core version (v12.X).

Reply:
Totally fine! Thanks for fixing it up!